flight/drone/drone.py

```diff
@@ -205,3 +211,16 @@ def _make_attitude_message(self, roll, pitch, yaw, thrust):
             radians(yaw), # Body yaw rate in radians
             thrust  # Thrust
         )
+
+    def _make_yaw_message(self, heading, yaw_speed=0, yaw_direction=1,relative=False):
+        return self.message_factory.command_long_encode(
+            0, 0,    # target system, target component
+            mavutil.mavlink.MAV_CMD_CONDITION_YAW, #command
+            0, #confirmation
+            heading,    # param 1, yaw in degrees
```

Comment:
what is this comment whitespace. make sure you're consistently using spaces and not tabs in your editor.